ze/sampling_daemon.c

```diff
@@ -768,23 +770,24 @@ int main(int argc, char **argv) {
     _DL_ERROR_MSG();
     return 1;
```

Comment:
I think this will deadlock as we the host will wait for `ready`. I didn't try, maybe we are luck as the non-zero exit code will trigger something and everybody will be happy and bail.